editor/export/editor_export_preset.cpp

```diff
@@ -332,4 +334,32 @@ Variant EditorExportPreset::get_or_env(const StringName &p_name, const String &p
 	return get(p_name, r_valid);
 }
 
+String EditorExportPreset::get_version(const StringName &p_preset_string, bool p_windows_version) const {
+	String result = get(p_preset_string);
+	if (result.is_empty()) {
+		result = GLOBAL_GET("application/config/version");
+
+		if (p_windows_version) {
+			// Modify version number to match Windows constraints (version numbers must have 4 components).
+			const PackedStringArray result_split = result.split(".");
+			String windows_version;
+			if (result_split.is_empty()) {
+				// Use a valid fallback if the version string is empty, as a version number must be specified on Windows.
```

Comment:
What about other platforms? Is it fine to return an empty string for them?

Reply:
No, we should also return a fallback for those in case the version project setting is set to an empty string.